jest.config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "moduleDirectories": ["node_modules", "src"],
+  "moduleNameMapper": {
+    "@/components/(.*)": "<rootDir>/src/components/$1",
+    "@/utils/(.*)": "<rootDir>/src/utils/$1"
+  },
+  "setupFilesAfterEnv": ["<rootDir>/setupTests.ts"]
+}
```

Comment:
We already have some jest config in the `package.json`. It think we just should use 1 place for the config. So either `package.json` or the `jest.config.json`

Reply:
I guess it'd be better if we separate the Jest config into its own file. It's more predictable that way.